README.md

```diff
@@ -422,10 +422,20 @@ cekit build --overrides '{"version": "SNAPSHOT", "artifacts": [{"name": "config-
 image.
 
 ### Data Grid
-In order to create an image using the Red Hat Data Grid server, it's necessary to have an active Red Hat kerberos session. The image can then be created using the following command:
+In order to create an image using the Red Hat Data Grid server, it's necessary to have an active Red Hat kerberos session.
+
+To build the OpenJDK (HotSpot) image (x86_64):
+```bash
+cekit build --overrides-file dg-override.yaml [docker|podman|osbs|...]
+```
+
+To build the OpenJ9 image (s390x):
 ```bash
-cekit build --overrides-file dg-override.yaml docker
+cekit build --overrides-file dg-override.yaml --overrides-file openj9-override.yaml [docker|podman|osbs|...]
```

Comment:
This is not totally true, s390x/OpenJ9 only works using `osbs` engine, because locally (`docker`, `podman`, ...) we don't have the OpenJ9 packages available. Maybe we should change the command line and add a note about it?

Reply:
I see. I'll update it